core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -1203,9 +1203,9 @@ private[spark] object SparkSubmitUtils {
   def resolveDependencyPaths(
       artifacts: Array[AnyRef],
       cacheDirectory: File): Seq[String] = {
-    artifacts.map { artifactInfo =>
-      val artifact = artifactInfo.asInstanceOf[Artifact].getModuleRevisionId
-      val extraAttrs = artifactInfo.asInstanceOf[Artifact].getExtraAttributes
+    artifacts.map(_.asInstanceOf[Artifact]).filter(_.getExt == "jar").map { artifactInfo =>
```

Comment:
Is it worth showing a list of non-jar files that are filtered out here (e.g., `logInfo` or `logWarning`)?

Reply:
+1

Reply:
Ya, maybe, `logInfo`.

Reply:
Added this logging

`21/03/05 06:40:48 INFO SparkSubmitUtils: Skipping non-jar dependency org.apache.curator#apache-curator;2.7.1!apache-curator.pom`